person.rb

```diff
@@ -53,10 +53,3 @@ def correct_name
     @nameable.correct_name[0...10]
   end
 end
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227525029-0459210f-1290-4bf2-bb18-2b90a912907a.png)

**[OPTIONAL]**

- [ ] Kindly add a the actual tech stack which has been used in this project to make your readme file descriptive 👍